### PR TITLE
Changed legacy SMART plugins to Vonk.Plugin.Smart

### PR DESCRIPTION
**Pull request by Sieuwkje-firely**

### CODE REVIEW

File: releasenotes/releasenotes_v6.rst [hunk 1]
```diff
@@ -20,6 +20,7 @@ Features
 ^^^^^^^^
 
 #. Firely Server offers the ``$member-match`` operation to find the identifier of a member of a health plan. See :ref:`member-match` for more information.
+#. Starting from this release the ``Vonk.Smart`` and ``Vonk.Plugin.SoFv2`` plugins are no longer supported. They are replaced by the ``Vonk.Plugin.Smart`` plugin. For more information see :ref:`feature_accesscontrol_config`.
```
Comment (alexzautke) [inline releasenotes/releasenotes_v6.rst:23]:
```suggestion
#. Starting from this release the ``Vonk.Smart`` and ``Vonk.Plugin.SoFv2`` plugins are no longer supported and have been removed. They are replaced by the ``Vonk.Plugin.Smart`` plugin. For more information see :ref:`feature_accesscontrol_config`. It is necessary to adjust the pipeline options accordingly.
```

File: security/smart.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@ configuration in the ``PipelineOptions`` section, or copy that section from ``ap
 Add ``Vonk.Plugin.Smart`` to the list of included plugins. When you restart Firely Server, the Smart service will be added to the pipeline.
 
 .. note:: 
-  From Firely Server v5.5.0 on, the ``Vonk.Plugin.Smart`` plugin replaces the legacy plugins ``Vonk.Smart`` (for SMART v1) and ``Vonk.Plugin.SoFv2`` (for SMART v2). For Firely Server versions older than v5.5.0, these legacy plugins may still be used. 
+  From Firely Server v5.5.0 on, the ``Vonk.Plugin.Smart`` plugin replaces the legacy plugins ``Vonk.Smart`` (for SMART v1) and ``Vonk.Plugin.SoFv2`` (for SMART v2). For Firely Server versions older than v5.5.0, these legacy plugins may still be used. For Firely Server versions v6.0.0 and newer, the legacy plugins are no longer supported.
```
Comment (alexzautke) [inline security/smart.rst:36]:
```suggestion
  From Firely Server v5.5.0 on, the ``Vonk.Plugin.Smart`` plugin replaces the legacy plugins ``Vonk.Smart`` (for SMART v1) and ``Vonk.Plugin.SoFv2`` (for SMART v2). For Firely Server versions older than v5.5.0, these legacy plugins may still be used. For Firely Server versions v6.0.0 and newer, the legacy plugins are no longer supported and have been removed.
```

**Review by alexzautke — COMMENTED** [review]
Added two suggestions

**Comment by Sieuwkje-firely** [comment]
Comitted them, thanks!

**Review by alexzautke — APPROVED** [review]

**Pull request merged**